docs/flank-output-investigation/flank_output_propositon.md

```diff
@@ -0,0 +1,101 @@
+# Flank output proposition
+
+## Android
+
+### Version section
+
+```bash
+
+version: local_snapshot
+revision: d5e7c76ab206373c6cdf0c1a25e2575f323e5929
+session id: 7a7ceda1-658e-461f-bd37-04f2b2a90a37
+
+```
+
+### Args section
+
+```bash
+<Diff of arguments>
```

Comment:
What does it mean? Flank will print only those arguments that differ from default ones?

Reply:
Diff of arguments or diff of output?

Reply:
It will print arguments provided by user, so differ from default ones

Reply:
Is this something new, or we are displaying this yet?

Reply:
For now, we are displaying all arguments (default and custom ones), but yesterday we spoke with Adam and we come up with the conclusion that this just make output less readable